py3-cython.yaml

```diff
@@ -38,15 +38,47 @@ subpackages:
     name: py${{range.key}}-${{vars.pypi-package}}
     dependencies:
       provider-priority: ${{range.value}}
-      provides:
```

Comment:
don't we want to
```yaml
dependencies:
  provides:
    - py3-${{vars.pypi-package}}
```

Reply:
I do!

Reply:
Do people expect `py3-cython` to provide `cython`?